profiling.md

```diff
@@ -38,8 +38,6 @@ If using GASNet, ensure that [GASNet has been properly configured and
 optimized]({{ "/gasnet/" | relative_url }}). GASNet can be important for performance on
 multi-node runs.
 
-## Legion Prof
```

Comment:
Please be sure to go to the top of this file to update the table of contents.

Reply:
Done.